src/printer.ts

```diff
@@ -1572,11 +1574,12 @@ export class PugPrinter {
 	}
 
 	private each(token: EachToken): string {
+		const code: string = format(token.code, { parser: '__js_expression', ...this.codeInterpolationOptions });
```

Comment:
Maybe we want to call `formatDelegatePrettier` here instead of doing it manually? 🤔

Reply:
or should it be `frameworkFormat`?

Reply:
You mean the name of the function? :thinking: 
I named it to delegate the formatting with/to prettier
It's not the framework but somewhat like an underlying/embedded format call

Reply:
like instead of `formatDelegatePrettier`. it doesn't look like `formatDelegatePrettier` actually takes in to account what the framework is.

Reply:
Have a look into L603 - L657 :slightly_smiling_face: 
it is only taking the requested formatter, the formatter could be a framework but also could be just e.g. json or css

Reply:
you could have something like `each item in "{{ array }}"` in angular for example.

Reply:
I would anyway suggest to not change it in this PR, cause it's unrelated :slightly_smiling_face:

Reply:
that was kind of what i was starting to think.